pom.xml

```diff
@@ -21,6 +21,8 @@
     <module>java-common-protos</module>
     <module>java-iam</module>
     <module>gapic-generator-java-bom</module>
+    <module>showcase</module>
```

Comment:
Does this have any impact on the recommended workflows in `DEVELOPMENT.md`? (Particularly, are there going to be any issues with the `mvn install` command -- or will showcase testing largely just be skipped? Will this add any significant delay to the development cycle now that Showcase is being generated, or will generation not be a burden?)

Reply:
Sounds great, thank you.